Codecademy/gamut
# fix(SelectDropDown): fix option voice over announcement bug

### Overview





### PR Checklist

- [x] Related to JIRA ticket: [EN-2091](https://codecademy.atlassian.net/browse/EN-2091)

#### Testing Instructions



1. Visit https://caiman.codecademy.com//admin and impersonate user: ``test-tracking-admin@testerv.com`` 
2. Visit https://caiman.codecademy.com/learn/learn-python-3 and click on the CTA ``Assign Course``
3. Enable voice over and Tab navigate to the select drop down input box
4. use to the up and down arrows keys to navigate
5. Ensure that when you land on the disabled user you hear the ``Already Assigned`` message

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/34100)  | [Monolith Env](https://caiman.codecademy.com/learn/learn-python-3) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/959)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-959')   |





[EN-2091]: https://codecademy.atlassian.net/browse/EN-2091?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
